test.js

```diff
@@ -14,8 +14,10 @@ const set = require('./');
 
 describe('set', function() {
   it('should return non-objects', function() {
-    const res = set('foo', 'a.b', 'c');
+    var res = set('foo', 'a.b', 'c');
```

Comment:
better to use `let` than `var` to avoid hoisting.

Reply:
As I see the file has `const` and `var` declarations only, so I've used `var` for consistency

Reply:
fair enough